doc/source/tutorial/tutorial_sessions.ipyml

```diff
@@ -147,44 +141,36 @@ cells:
 
 
 - code: |
-    # define a function to add an extra year
-    def add_extra_year(array):
-        last_year = array.time.i[-1]
-        last_value = array[last_year]
-        # add an extra year to the time axis and 
-        # duplicate the value from the last year
-        return array.append('time', last_value, label=last_year+1)
+    # define a function convert a float array as
+    # an int array
+    def as_type_int(array):
+        return array.astype(int)
     
-    s_pop_extra_year = s_pop.apply(add_extra_year)
+    s_pop_rand_int = s_pop_rand.apply(as_type_int)
     
     print('pop array before calling apply:')
-    print(s_pop.pop)
+    print(s_pop_rand.pop)
     print()
     print('pop array after calling apply:')
-    print(s_pop_extra_year.pop)
+    print(s_pop_rand_int.pop)
 
 
 - markdown: |
     It is possible define a function with additional arguments:
 
 
 - code: |
-    # define a function to add extra years
-    def add_extra_years(array, nb_years):
-        last_year = array.time.i[-1]
-        last_value = array[last_year]
-        # add extra years to the time axis
-        time = array.time.extend(range(last_year+1, last_year+nb_years+1))
-        # duplicate the value from the last year
-        return array.reindex('time', time, fill_value=last_value)
+    # define a function to change the dtype of an array
```

Comment:
FWIW, one need not define a new function for that. we can simply use .apply(LArray.astype, dtype=int). Defining a new explicit function is a good idea for the sake of the example but either it should be a different example (a function which does not already exists) or a comment should be present to say that we define the function for the sake of the example.